sci-libs/pytorch/pytorch-1.10.1.ebuild

```diff
@@ -230,6 +230,7 @@ src_prepare() {
 		export PYTORCH_ROCM_ARCH="${AMDGPU_TARGETS}"
 		sed -e "/set(roctracer_INCLUDE_DIRS/s,\${ROCTRACER_PATH}/include,${EPREFIX}/usr/include/roctracer," \
 			-e "/PYTORCH_HIP_HCC_LIBRARIES/s,\${HIP_PATH}/lib,${EPREFIX}/usr/lib/hip/lib," \
+			-e "/set(roctracer_INCLUDE_DIRS/a\  set(thrust_INCLUDE_DIRS ${EPREFIX}/usr/include/rocthrust)" \
```

Comment:
Does this line make line 231 obsolete? Or do we need both of them?

Reply:
We will need both of them the `set(roctracer_INCLUDE_DIRS` is just to add `"/set(roctracer_INCLUDE_DIRS/a\  set(thrust_INCLUDE_DIRS ${EPREFIX}/usr/include/rocthrust)` to the right line. Not sure if patch wouldn't be more appropriate.